<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreQuery</class>
 <widget class="QDialog" name="ScoreQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>應修學分統計</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ScoreQuery{
background-color: rgb(255, 252, 235);
}

    QPushButton{
		border:0;
		font: 700 11pt ;
		border:1px solid #000;
		background:#fff;
		border-radius:10px;
    }
    QPushButton:hover{
        border:2px solid #000;
	
	background-color: rgb(255, 253, 188);
    }</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>721</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;微軟正黑體&quot;;
background-color: rgb(231, 255, 241);</string>
   </property>
   <property name="markdown">
    <string>**應修學分:{Score}/128 學分 **

**校(選擇性)必修、院訂必修:{point1}/36 學分 **

**院、系專業必修:{point2}/68 學分 **

**系專業選修:{point3}/24 學分 **

**通職:{pointS1}/4 通過項目 (已過:{pointS1BlackList})**

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微軟正黑體'; font-size:10pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;應修學分:{Score}/128 學分 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;校(選擇性)必修、院訂必修:{point1}/36 學分 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;院、系專業必修:{point2}/68 學分 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;系專業選修:{point3}/24 學分 &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft JhengHei UI'; font-size:12pt;&quot;&gt;通職:{pointS1}/4 通過項目 (已過:{pointS1BlackList})&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>721</width>
     <height>341</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;微軟正黑體&quot;;
background-color: rgb(231, 255, 241);</string>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>721</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="sz">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="btnPointAll">
      <property name="text">
       <string>所有分數</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPoint1">
      <property name="text">
       <string>校(選擇性)必修、院訂必修</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPoint2">
      <property name="text">
       <string>院、系專業必修</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPoint3">
      <property name="text">
       <string>系專業選修</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUnPass">
      <property name="text">
       <string>未通過</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="filterText">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>篩選科目名稱</string>
   </property>
   <property name="styleSheet">
    <string notr="true">
		border:0;
		font: 700 11pt ;
		border:1px solid #000;
		background-color: rgb(221, 221, 221);
		border-radius:10px;
		td align=&quot;center&quot; valign=&quot;center&quot;;
		font: 8pt &quot;微軟正黑體&quot;;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="placeholderText">
    <string>篩選科目名稱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGlobalPassRate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>正在連結...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
